spec/files/osws/prototype_suite.osw

```diff
@@ -1,11 +1,11 @@
 {
 	"seed_file": null,
-	"weather_file": null,
+	"weather_file": "srrl_2013_amy.epw",
 	"measure_paths": [
     "../../files/measures",
     "../../../lib/measures"
   ],
-	"file_paths": ["../../files/"],
+	"file_paths": ["../../files/epws", ""],
```

Comment:
@macumber: When running prototype test suite, noticed missing .osm files in generated folders (except "skip"). So my changes are breaking workflows. Will take another look soonish, but if anything obvious  pops up ...